pkg/database/migrations.go

```diff
@@ -2396,6 +2396,21 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 						DROP COLUMN IF EXISTS headless`)
 			},
 		},
+		{
+			ID: "00112-AddUserReportLearnMore",
+			Migrate: func(tx *gorm.DB) error {
+				return multiExec(tx,
+					`ALTER TABLE realms
+						ADD COLUMN IF NOT EXISTS user_report_learn_more TEXT`,
```

Comment:
Nit: this is a link right? It'd be good to name it *Link

Reply:
added URL to the field and DB column